app/src/components/Chat.tsx

```diff
@@ -23,45 +24,83 @@ export default function Chat({
   const [conversationId, setConversationId] = useState<number | null>(null);
   const [waitingForResponse, setWaitingForResponse] = useState(false);
```

Comment:
Still need to split into two... to avoid having both dots and streaming text

![image](https://github.com/adamcohenhillel/ADeus/assets/19780885/f244a98d-f014-4f65-8571-4d0d5a145593)

Reply:
done